forge-gui/res/cardsfolder/upcoming/death_rattle_oni.txt

```diff
@@ -0,0 +1,10 @@
+Name:Death-Rattle Oni
+ManaCost:6 B
+Types:Creature Demon Spirit
+PT:5/4
+K:Flash
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {2} less to cast for each creature that died this turn.
+SVar:X:Count$ThisTurnEntered_Graveyard_from_Battlefield_Creature/Times.2
+T:Mode$ ChangesZone | TriggerZone$ Battlefield | ValidCard$ Card.Self | Origin$ Any  | Destination$ Battlefield | Execute$ DBDestroyAll | TriggerDescription$ CARDNAME enters the battlefield, destroy all other creatures that were dealt damage this turn.
+SVar:DBDestroyAll:DB$ DestroyAll | ValidCards$ Creature.wasDealtDamageThisTurn+OppCtrl
```

Comment:
+OppCtrl needs to be removed

Reply:
Thanks for your review comment but I think you are reviewing an older version